chainer_chemistry/dataset/preprocessors/rsgcn_gwm_preprocessor.py

```diff
@@ -0,0 +1,67 @@
+from chainer_chemistry.dataset.preprocessors.common import construct_adj_matrix  # NOQA
+from chainer_chemistry.dataset.preprocessors.common import construct_atomic_number_array  # NOQA
+from chainer_chemistry.dataset.preprocessors.common import type_check_num_atoms  # NOQA
+from chainer_chemistry.dataset.preprocessors.common import construct_supernode_feature # NOQA
+from chainer_chemistry.dataset.preprocessors.mol_preprocessor import MolPreprocessor  # NOQA
+
+
+import numpy
+
+
+class RSGCNGWMPreprocessor(MolPreprocessor):
```

Comment:
It is ok for now, but we might refactor to make it subclass of RSGCNPreprocessor to reduce redundant code.
Same applies for other preprocessors.